### ISSUE TITLE
VGG16-DDT GT Known Loc Accuracy on CUB200

### ISSUE THREAD

**Issue by qinwang-ai** [tzzcl/PSOL#7]
I modified 'generate_box_imagenet.py' to 'generate_box_cub200.py'. However, I got the accuracy of GT known Loc is 88.5% which is much higher than paper claimed 84.55% in Tab.1. Cloud you please give some explanations ? or release your own 'generate_box_cub200.py'.

Thanks,

**Comment by tzzcl** [comment]
Hi, Thank you for your interest in our project!

Actually, when I do experiments  in this paper, I first do the CUB experiment in late 2018, then I just do the ImageNet experiment in 2019. So the CUB results may be somehow old. Besides that part, I found that upgrading the PyTorch version will also improve the performance (I remember that PSOL can have 1%+ GT-Known acc improvement even we do not change the box and any hyperparameters). I guess results on CUB may also benefit from this phenomenon. So the performance improvement may come from this side, though I can not found detailed reasons. Maybe it comes from the network (pre-trained models side) or it comes from other packages.

But I'm happy that DDT can get better results, then PSOL can get better results.

Thanks.

**Comment by qinwang-ai** [comment]
@tzzcl Thanks. Could you please give the GT-known Loc Acc on CUB200 of PSOL ? I cannot find this column in Table 2 of the paper.
![image](https://user-images.githubusercontent.com/5886719/95718326-dd2dd080-0ca0-11eb-8de5-04025deb8ed8.png)

**Comment by tzzcl** [comment]
> @tzzcl Thanks. Could you please give the GT-known Loc Acc on CUB200 of PSOL ? I cannot find this column in Table 2 of the paper.
> ![image](https://user-images.githubusercontent.com/5886719/95718326-dd2dd080-0ca0-11eb-8de5-04025deb8ed8.png)

Can you give me your email address? I will send some results and explanations to you by email.

**Comment by tzzcl** [comment]
Can you give me your email? I will send to you some details and results.

Thanks

**Comment by qinwang-ai** [comment]
@tzzcl My Email: qinwang1@link.cuhk.edu.cn

Thank you very much!

**Comment by Jazzcharles** [comment]
@tzzcl @qinwang-ai  
I'm working on CUB200 too. Can you please share the details and results? Thank you very much !!! 
My email:  18210240039@fudan.edu.cn
